tests/test_masks.py

```diff
@@ -47,6 +47,13 @@ def test_init(self):
                 layers=size,
                 wires=size + 1,
             )
+        mc = MaskedCircuit(
+            parameters=pnp.random.uniform(low=0, high=1, size=(size, size)),
+            layers=size,
+            wires=size,
+            wire_mask=pnp.array([True] * size),
+        )
+        assert pnp.array_equal(mc.wire_mask, [True] * size)
```

Comment:
You might want to try the approaches in [How to create a numpy array of all True or all False?](https://stackoverflow.com/q/21174961/5349916). Haven't benchmarked them, but I assume they are faster in this case.

Reply:
Note that I'm fine either way – the ``[True] * size`` is pretty legible and test performance should not matter.